bs-emotion-ppx/postinstall.js

```diff
@@ -0,0 +1,28 @@
+var os = require('os')
+var fs = require("fs");
+var path = require("path");
+
+var rootDir = __dirname;
+var binDir = path.join(rootDir, "bin");
+
+/**
+ * Returns OS specific binary to copy to "bin" folder.
+ */
+function getBinaryDetails() {
+    switch (os.type()) {
```

Comment:
We should check the arquitecture as well and warn users in unsupported platforms, e.g. https://github.com/jchavarri/hello-ppx-esy/blob/master/copyPlatformBinaryInPlace.js

Reply:
ok. Will push a new commit to address the issues in the next few days.